book/README.md

```diff
@@ -1,48 +1,47 @@
 # TLG Catalog
 
-Catalog for tables, listings, and graphs generated from NEST R packages.
+The catalog for (T)ables, (L)istings, and (G)raphs for clinical trials generated using NEST R packages.
 
-## Architecture
+The main goal of this repository is to provide a comprehensive collection of clinical trials outputs generated using R packages. The target audience is the clinical trials community, including statisticians, data scientists, and other professionals interested in applying R to clinical trials data.
 
-This repository follows a very straightforward architecture.
+## Usage
 
-``` mermaid
-sequenceDiagram
-    autonumber
-    Author->>github.com: Creates content + pushes updates
-    github.com->>Github Pages: Builds and publishes
-    Analyst->>Github Pages: Refers to catalog
-```
+Each of the output is generated in separate articicle and usually consists of:
 
-## Technology
+* Data setup using dummy, synthetic data.
+* Output itself (of many variants if applicable).
+* Interactive application.
```

Comment:
Yes. This is a single module teal with appropriate arguments values so that it corresponds to the particular TLG definition.

Technically it's a R code that Quarto will parse, eval and do a screeshot. In the near future I hope we will experiment with WebR to make it an actual app. CRAN release is the first blocker. But that's future.